servers/lib/package.json

```diff
@@ -10,72 +10,77 @@
   ],
   "private": false,
   "license": "SEE LICENSE IN <LICENSE.md>",
+  "type": "module",
+  "main": "dist/src/main.js",
   "scripts": {
     "build": "tsc",
     "clean": "npx rimraf build node_modules coverage dist src.svg test.svg",
     "format": "prettier --ignore-path ../.gitignore --write \"**/*.{ts,tsx,css,scss}\"",
     "graph": "npx madge --image src.svg src && npx madge --image test.svg test",
-    "start": "node dist/src/main.js",
-    "start:pm2": "pm2 start pm2.config.js",
+    "start": "tsc & node dist/src/main.js",
+    "start:pm2": "pm2 start pm2.config.cjs",
     "stop:pm2": "pm2 delete libms",
     "syntax": "eslint . --fix",
```

Comment:
Gives the following error (only on Windows):

```bash
$ yarn syntax
yarn run v1.22.19
$ eslint . --fix

Oops! Something went wrong! :(

ESLint: 9.11.1

ConfigError: Config (unnamed): Unexpected key "0" found.
    at rethrowConfigError (C:\Users\user\git\DTaaS\servers\lib\node_modules\@eslint\config-array\dist\cjs\index.cjs:303:8)
    at C:\Users\user\git\DTaaS\servers\lib\node_modules\@eslint\config-array\dist\cjs\index.cjs:1098:5
    at Array.reduce (<anonymous>)
    at FlatConfigArray.getConfigWithStatus (C:\Users\user\git\DTaaS\servers\lib\node_modules\@eslint\config-array\dist\cjs\index.cjs:1091:43)
    at FlatConfigArray.getConfig (C:\Users\user\git\DTaaS\servers\lib\node_modules\@eslint\config-array\dist\cjs\index.cjs:1120:15)
    at C:\Users\user\git\DTaaS\servers\lib\node_modules\eslint\lib\eslint\eslint-helpers.js:354:54
    at C:\Users\user\git\DTaaS\servers\lib\node_modules\eslint\lib\eslint\eslint-helpers.js:296:32
    at Object.isAppliedFilter (C:\Users\user\git\DTaaS\servers\lib\node_modules\@nodelib\fs.walk\out\readers\common.js:12:31)
    at AsyncReader._handleEntry (C:\Users\user\git\DTaaS\servers\lib\node_modules\@nodelib\fs.walk\out\readers\async.js:86:20)
    at C:\Users\user\git\DTaaS\servers\lib\node_modules\@nodelib\fs.walk\out\readers\async.js:65:22
error Command failed with exit code 2.
```

Reply:
I don't get that error